### PR TITLE
Fix package command problems

### PR DESCRIPTION
**Pull request by florian-richter**

### CODE REVIEW

File: test/package.spec.ts [hunk 1]
```diff
@@ -2,20 +2,50 @@
...
   });
 
-  it('should copy files correctly with custom globs', () => {
-    expect(true).toBeTruthy();
+  it('should copy files correctly without parameters', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'no-params');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json']);
```
Comment (mr-flannery) [inline test/package.spec.ts]: From my understanding this tests the default value for `--include`. Shouldn't this test assert then that all files included by default are copied (i.e. package.json, package-lock.json, index.js, dist/**/*)?

File: test/package.spec.ts [hunk 2]
```diff
@@ -2,20 +2,50 @@
...
   });
 
-  it('should overwrite output folder', () => {
-    expect(true).toBe(true);
+  it('should copy files correctly with custom globs', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'globs');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--include=*.json', '--exclude=package*,tsconfig*', '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['nest-cli.json', 'tslint.json']);
```
Comment (mr-flannery) [inline test/package.spec.ts]: Similar to the comment above, if you set the `--exclude` flag, I would've expected the test to also assert that the excluded files are not present in the target directory.

File: test/package.spec.ts [hunk 3]
```diff
@@ -2,20 +2,50 @@
...
 
-  it('should install productive dependencies only', () => {
-    expect(true).toBe(true);
+  it('should overwrite output folder', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'folder-overwrite');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--include=.gitignore', '--skipInstall']);
+    await Package.run([projectDir, '--include=README.md', '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toEqual(['README.md']);
```
Comment (mr-flannery) [inline test/package.spec.ts]: As above, I would've expect the test to assert that `.gitignore` does not exist in the output directory.

Comment (florian-richter) [reply test/package.spec.ts]: This test uses `.toEqual()` and therefore check if the directory ONLY includes the `README.md`

File: test/package.spec.ts (hunk 1 above)
Comment (florian-richter) [reply test/package.spec.ts]: for the default nest.js app does not have an `index.js` and the `dist/**` only appear after compiling (which I dont test here). That's why I implemented it this way, but I am open for improvement suggestions.

File: test/package.spec.ts (hunk 3 above)
Comment (mr-flannery) [reply test/package.spec.ts]: Ah, that escaped me, thanks!

File: test/package.spec.ts (hunk 1 above)
Comment (mr-flannery) [reply test/package.spec.ts]: I see. Is there something like `toIncludeAndOnlyIncludeAllMembers`? =P

**Pull request merged**